docs/b2b-edition/specs/storefront/storefront/company.yaml

```
@@ -783,8 +785,8 @@ paths:
                         email: example@example.com
       operationId: get-companies-validations-fronted-user-emails-email
```

```suggestion
      operationId: get-companies-validations-storefront-user-email
```

looks like this has typos, and instead of `frontend` we should talk about this as a `storefront` operation

---

@bc-micah The endpoint shows  "/companies/validations/fronted/user-emails/{email}"
Is this correct? So I would need to change the URL, summary on line 697, operationId and description from front-end to storefront? Please clarify if the endpoint URL will change.

---

@bc-micah looking at the B2B API codebase, it looks like the endpoint in the swagger file is accurate (it's misspelled) so we'll leave it like this for now. If you create a new alias for the endpoint that is storefront, we can update!